pkg/kubernetes/scripts/listing.js

```diff
@@ -62,21 +62,21 @@
                     }
                 };
 
-                self.toggle = function toggle(id, ev) {
+                self.toggle = function toggle(id, kind, ev) {
```

Comment:
Can we make kind part of the id? That would be consistent with how we do it in the rest of the kubernetes pages.